lib/db/schema/patch-083-084.sql

```diff
@@ -0,0 +1,59 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('83');
+
+-- Since we are altering the size of the `recoveryKeyId` column
+-- we need to make sure that the column is empty before it can be applied.
+-- This table to see to clear at this point because no user facing api
```

Comment:
nit: typo here with " to see to clear"? sounds like it should be something like "is safe to clear".